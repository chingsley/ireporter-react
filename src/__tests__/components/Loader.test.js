import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loader from '../../components/Loader';


describe('Test the report form', () => {
  const shallowWrapper = shallow(<Loader />);
  it('should render with the correct elements', () => {
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
