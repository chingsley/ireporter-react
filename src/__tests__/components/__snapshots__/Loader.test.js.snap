// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the report form should render with the correct elements 1`] = `
<div
  className="loader"
>
  <img
    alt="loading gif"
    src="https://i.imgur.com/S39IaLt.gif"
  />
</div>
`;
